<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Inpatient</class>
 <widget class="QDialog" name="Inpatient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1064</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>220</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Department</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>260</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Doctor</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>300</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Time</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>340</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Type Room</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_InDepartment">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>220</y>
     <width>191</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_InDoctor">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>260</y>
     <width>191</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_InRoomType">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>340</y>
     <width>191</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>300</y>
     <width>194</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_InBook">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>450</y>
     <width>371</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Book Reservation</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>390</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>390</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Price:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>90</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>In-Patient</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
